Scripts/Runtime/Collections/NativeArrayExtension.cs

```diff
@@ -0,0 +1,62 @@
+using UnityEngine;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+
+
+/// <summary>
+/// Extension methods for use with a <see cref="NativeArray{T}"/> instance.
+/// </summary>
+public static class NativeArrayExtension
```

Comment:
Is it `Extension` or `Extensions`?

We seem to be using both? Doesn't matter to me, just clarifying.

Reply:
Where are we using Extensions? We aren't as far as I know.
Some 3rd party libraries that we depend on do add the `s`